src/components/join-flow/username-step.jsx

```diff
@@ -170,6 +174,9 @@ class UsernameStep extends React.Component {
                                         {this.props.intl.formatMessage({id: 'registration.choosePasswordStepTitle'})}
                                     </div>
                                     <FormikInput
+                                        autoCapitalize="off"
+                                        autoComplete={values.showPassword ? 'off' : 'new-password'}
```

Comment:
Why is there a distinction between new-password and off? Why wouldn't it be new-password regardless of whether the password is visible?

Reply:
It's because `showPassword` is also used by this input to determine whether it is `type="password"` or `type="input"` (really, just the default). 

AFAI can tell, "new-password" is the value that browsers look for if the input is type="password", and "off" is the value that browsers look for if the input is type="input".

Reply:
Cool. Thanks for explaining!